app/src/main/java/org/fossasia/openevent/general/event/EventsRecyclerAdapter.kt

```diff
@@ -34,6 +34,10 @@ class EventsRecyclerAdapter : RecyclerView.Adapter<EventViewHolder>() {
         this.events.addAll(eventList)
     }
 
+    fun removeAll(boolean: Boolean) {
```

Comment:
Name the variable properly. You don't name String string

Reply:
okay

Reply:
done